app/lib/efile/md/md502cr_calculator.rb

```diff
@@ -52,10 +56,6 @@ def calculate_md502_cr_part_m_line_1
           if @intake.primary_senior?
             credit = 1750
           end
-        elsif (filing_status_single? || filing_status_mfs? || md_filing_status_dependent?) && agi <= 100_000
-          if @intake.primary_senior?
-            credit = 1000
-          end
```

Comment:
again, for `md_filing_status_dependent` filer, they might be mfj / qw / hoh (+ have the necessary agi) and get the wrong credit amount without this refactor.

I think so far this probably hasn't happened b/c we haven't had a non-single dependent filer